roles/example-cnf-catalog/tasks/main.yml

```diff
@@ -89,3 +79,28 @@
       spec:
         targetNamespaces:
           - "{{ cnf_namespace }}"
+
+- name: Wait for testpmd-operator packagemanifests availability
+  k8s_info:
```

Comment:
let's use the FQCN, i.e `community.kubernetes.k8s_info`, here and all the other tasks that you're modifying

Reply:
Done